### PR TITLE
some pep8 fixes in symbolic (E305 and others)

### PR DESCRIPTION
**Pull request by fchapoton**
### :books: Description

This fixes E305 in symbolic folder, and most other pep8 warnings in the modified files.

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/symbolic/constants.py [hunk 1]
```diff
@@ -806,7 +810,7 @@ def __float__(self):
             sage: golden_ratio.__float__()
             1.618033988749895
         """
-        return float(0.5)*(float(1.0)+math.sqrt(float(5.0)))
+        return 0.5 + math.sqrt(1.25)
```
Comment (dcoudert) [inline src/sage/symbolic/constants.py:813]: This is not exactly the same value. Isn't it a problem ?
```py
sage: float(0.5)*(float(1.0)+math.sqrt(float(5.0)))
1.618033988749895
sage: 0.5 + math.sqrt(1.25)
1.61803398874989
sage: float((1 + sqrt(5))/2)
1.618033988749895
```
But we have
```py
sage: float(0.5)*(float(1.0)+math.sqrt(float(5.0))) == 0.5 + math.sqrt(1.25)
True
```

Comment (fchapoton) [reply src/sage/symbolic/constants.py:813]: The preparser is not active in source files:
```
sage: preparser(False)
sage: sage: float(0.5)*(float(1.0)+math.sqrt(float(5.0)))
....: sage: 0.5 + math.sqrt(1.25)
....: sage: float((1 + sqrt(5))/2)
1.618033988749895
1.618033988749895
1.618033988749895
```

Comment (dcoudert) [reply src/sage/symbolic/constants.py:813]: Thanks for the clarification.

**Review by dcoudert — APPROVED** [review]
LGTM.

**Pull request merged**